graphql/src/store/query.rs

```diff
@@ -180,6 +181,7 @@ fn build_filter_from_object(
                     EndsWith => EntityFilter::EndsWith(field_name, store_value),
                     NotEndsWith => EntityFilter::NotEndsWith(field_name, store_value),
                     Equal => EntityFilter::Equal(field_name, store_value),
+                    // Child => EntityFilter::And(build_filter_from_object(..))
```

Comment:
Or instead of flattening the list of EntityFilters, we could return `EntityFilter:And` built out of the 2nd level input object.